### PR TITLE
feat(ui): preview table CSVW

### PR DESCRIPTION
**Pull request by martinmaillard**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3231,7 +3336,34 @@ balanced-match@^1.0.0:
...
+    "@rdfjs/formats-common" "^2.0.0"
+    duplexify "^3.6.1"
+    lodash "^4.17.15"
+    nodeify-fetch "^2.0.0"
+    rdf-ext "^1.1.2"
+    rdf-transform-triple-to-quad "^1.0.1"
+    readable-stream "^3.0.6"
+    through2 "^2.0.3"
+
+barnard59-base@^0.0.5:
```
Comment (tpluscode) [inline yarn.lock:3357]: Wonder why this happened. `package.json` resolutions should have prevented

**Review by tpluscode — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (martinmaillard) [reply yarn.lock:3357]: No clue

Comment (tpluscode) [reply yarn.lock:3357]: I thought that old barnard base is why pipelines swallow some errors. Unconfirmed so far

**Comment by tpluscode** [comment]
The failed e2e tests were not something to correct first?

**Pull request merged**